ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json

```diff
@@ -22,13 +19,11 @@
         "sources": [
           {"name": "OUTPUT", "type": "stream", "schema": "ID INT KEY, ROWPARTITION STRING, ROWOFFSET STRING"}
         ]
-      }
+      },
+      "comments": "Should be removed after rowpartition.rowoffset.enabled feature flag is removed"
```

Comment:
Almost but not quite -- when the feature flag is removed we'll actually want to remove the other versions of these tests (the newly added ones with `- ROWOFFSET/ROWPARTITION enabled` in the names).

Also, we typically put `comments` above `name` so that readers know to read the comments (typically containing info relevant for understanding the rest of the test) before actually reading the test.

Reply:
At the risk of sounding dumb: I'm confused, don't we want to ensure that when the flag is enabled we are disallowing these reserved names to be used? I thought the whole reason we added these tests was so we could add them and only persist the historical plans. Since this is the last change, I'll push the branch without any changes for now until I understand why this is the case.

Reply:
> don't we want to ensure that when the flag is enabled we are disallowing these reserved names to be used?

We do, but when the flag is removed then there won't be anything special about ROWPARTITION and ROWOFFSET anymore; they'll just be regular pseudocolumns like all the others. So when that happens, it won't make sense to have a test called "- ROWPARTITION and ROWOFFSET enabled". We'll have to update the regular versions of the tests to account for ROWPARTITION and ROWOFFSET and so it's just a matter of deciding which test name we want. We should keep the original since the name is shorter and we won't have to introduce a new historic plan folder.

Reply:
So we should change the functionality of the above test to be equivalent to  the `enabled` test after the flag is ungated? With a corresponding comment saying "should remove and change {other test} to reflect this behavior after feature flag is ungated" under the `-enabled` tests.

Reply:
We don't need to leave a comment on this test since we'll be forced to change it once the flag is ungated -- otherwise the test will fail. We just need to leave a comment on the other test saying it should be removed once the flag is ungated, else we'll have duplicate tests.